### ISSUE TITLE
Failed install?

### ISSUE THREAD

**Issue by Btibert3** [Pooja-Rajkumar/rectools#1]
What are the install requirements?  The error message below (when using `devtools`) as well as my system info.

```
> devtools::install_github("Pooja-Rajkumar/rectools")
Downloading GitHub repo Pooja-Rajkumar/rectools@master
from URL https://api.github.com/repos/Pooja-Rajkumar/rectools/zipball/master
Installing rectools
Installing 2 packages: lme4, partools
'/Library/Frameworks/R.framework/Resources/bin/R' --no-site-file --no-environ --no-save --no-restore  \
  CMD INSTALL  \
  '/private/var/folders/hv/wtvckymn0230hpsdwylmtf0r0000gn/T/RtmphZ1Cff/devtoolsdd285c2eb90e/Pooja-Rajkumar-rectools-9b11a16'  \
  --library='/Library/Frameworks/R.framework/Versions/3.2/Resources/library' --install-tests 

* installing *source* package ‘rectools’ ...
** R
Error in parse(outFile) : 
  /private/var/folders/hv/wtvckymn0230hpsdwylmtf0r0000gn/T/RtmphZ1Cff/devtoolsdd285c2eb90e/Pooja-Rajkumar-rectools-9b11a16/R/kN:26:36: unexpected symbol
25:    checkNewItem <- function(oneUsr) { tmp <- match(oneUsr$itms,newItem)
26:    if (all(is.na(tmp))) return(NA) whichOne
                                       ^
ERROR: unable to collate and parse R files for package ‘rectools’
* removing ‘/Library/Frameworks/R.framework/Versions/3.2/Resources/library/rectools’
Error: Command failed (1)
```

and my session

```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.11.4 (El Capitan)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] httr_1.2.0      R6_2.1.2        tools_3.2.3     withr_1.0.1     curl_0.9.7      memoise_1.0.0  
[7] git2r_0.12.1    digest_0.6.9    devtools_1.10.0
```

**Comment by matloff** [comment]
Oops!

That was an experimental file.  I've removed it now.  Please try again,
should install fine.

Sorry for the inconvenience.  We'd appreciate any feedback you have on
the package.

Norm

**Comment by Btibert3** [comment]
Thanks for the quick reply.  I was able to get it installed, but just as a heads up, it failed the first time around because I didn't have the package `lme4` installed.  After running `install.packages("lme4")` and then `devtools::install_github("Pooja-Rajkumar/rectools")`, everything worked.

**Comment by matloff** [comment]
That's odd.  The DESCRIPTION file specifies a dependency on lme4, so
install_github() should have noted the dependency and automatically
downloaded and installed lme4.

I just tried it myself.  I removed lme4 from my library directory, and
then tried to install rectools:

...

> devtools::install_github("Pooja-Rajkumar/rectools")
> Downloading github repo Pooja-Rajkumar/rectools@master
> Installing rectools
> Installing dependencies for rectools:
> lme4
> Installing package into ‘/home/matloff/Pub/Rlib’
> (as ‘lib’ is unspecified)
> --- Please select a CRAN mirror for use in this session ---
> ...

It then downloaded and installed lme4, and then did the same for
rectools.

Might it be that install_github() has some settings regarding this?  I
generally don't use devtools, so I can't say, but as seen above,
everything ran automatically when I tried it.

Norm
